### DISCUSSION TITLE
Sample server+dashboard applications with ASP.NET Core 6

### DISCUSSION THREAD

**Discussion by maiorfi** [elsa-workflows/elsa-core#2775] [Q&A] [answered]
It seems that following guide at https://elsa-workflows.github.io/elsa-core/docs/guides/guides-recurring-tasks but using ASP.NET Core 6 (and "no startup.cs" pattern using the new WebApplicationBuilder hosting model) causes some issue in designer hosting, that appears as "broken". Looking at resources requested by browser, I spotted that tailwind.css inclusion fails with 404.

Any hint?

**Comment by sfmskywalker** [comment, marked as answer]
Designer hosting should work just fine (I have a couple of projects running on .NET 6 here), so maybe something is missing with that sample in combination with .NET 6. We'll go through that guide & adapt it to .NET 6.

**Reply by maiorfi** [reply]
Thanks, very appreciated.

**Comment by emicortez** [comment]
I have the same problem, any update on this? Thanks in advance

**Reply by sfmskywalker** [reply]
I'm currently working on getting the designer to work for V3 and .NET 6 + .NET 7. Once V3 RC1 is out of the door, I'll try and allocate some time to revisit the V2 samples and documentation to update to .NET 6 and 7.

**Reply by emicortez** [reply]
good!! thanks

**Comment by Kawthar-Ebrahim** [comment]
Dear @sfmskywalker,
is the new version that support .NET6 available ? I tried using the latest version with .NET6 and faced some of errors, then I switched to use .NET5 just for experimenting Elsa and again I found some errors with the latest versions 
Such as "EventLog access is not supported on this platform." where I downgraded to a previous version (2.9.0) to solve this issue. 
I would like to know the current status and the plan to update the documentation.

**Reply by sfmskywalker** [reply]
Hi @Kawthar-Ebrahim 

I'm currently working on V3 while supporting customers with V2, which means my bandwidth to work on V2 documentation is pretty much non-existent at the moment. My plan is to update the V2 documentation as soon as V3 is out of the door (which is being heavily pushed by some of the aforementioned customers). As you might imagine, this makes it tricky to give a reliable planning, but I'll do my best to make this happen as soon as possible.

**Reply by zachion** [reply]
I was experiencing the same issue, "EventLog access is not supported on this platform." Downgrading to Elsa 2.9.0 worked. However I've also found updating project to .NET 7.0 & Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation 7.0.3 also fixed the above issue.

**Comment by MrkacekVladimir** [comment]
Hello,

currently I am trying to setup and make dashboard working. Currently I always get stuck at unstyled page because of tailwind.css missing (getting 404 but other files load normally)

Tried multiple versions: .NET 7 and Elsa versions 2.10, 2.9.3, 2.9.2, 2.9.0
Also tried .NET 5 and Elsa 2.9.3 with no success.

Any steps I need to take to eliminate the missing styles?

Thank you.

**Reply by MrkacekVladimir** [reply]
**Fixed**
For anyone having the same issue. The documentation has a major error in consistency.

As OP said, at this link the setup of _Host.cshtml is incorrect and contains invalid path to the stylesheet for Elsa.
https://elsa-workflows.github.io/elsa-core/docs/guides/guides-recurring-tasks 

Setting up dashboard + server should be done with this guide: https://elsa-workflows.github.io/elsa-core/docs/quickstarts/quickstarts-aspnetcore-server-dashboard-and-api-endpoints since there _Host file is updated to the correct version.
